File: InventoryManagement.Core/Entities/UnitOfMeasure.cs
using InventoryManagement.Core.Common;

namespace InventoryManagement.Core.Entities;

public class UnitOfMeasure : BaseEntity
{
    public string UnitName { get; set; } = string.Empty;
    public string? ShortName { get; set; }
    public string? Description { get; set; }

    // Navigation Properties
    public ICollection<Item> Items { get; set; } = new List<Item>();
}

File: InventoryManagement.UI/src/app/core/models/unit-of-measure.ts
export interface UnitOfMeasure {
  id: number;
  unitName: string;
  shortName?: string;
  description?: string;
  createdAt: Date;
}

export interface CreateUnitOfMeasureRequest {
  unitName: string;
  shortName?: string;
  description?: string;
}

export interface UpdateUnitOfMeasureRequest {
  unitName: string;
  shortName?: string;
  description?: string;
}